assets/js/main.js

```diff
@@ -24,6 +24,13 @@
         $('.show-menu').removeClass('show-outline');
       }
     };
+
+    // open the feedback form in the footer if on the website-feedbak page
+    if (window.location.href.indexOf("website-feedback") > -1) {
+      setTimeout(function(){ $('.reveal')[0].click()}, 100);
+      $('#sidebar-second').css('display', 'none');
+      $('#content').css('display', 'none');
```

Comment:
Hi Chris,
How about this:
```
setTimeout(function() { $('.reveal:first-child').click(); }, 100);
$('#sidebar-second').hide();
$('#content').hide();
```

Reply:
makes sense - i always steer away from using hide() but thinking about it i think this is learned behaviour and not grounded in reason ... updated thanks